bedrock/firefox/templates/firefox/nightly/whatsnew-fx145.html

```diff
@@ -0,0 +1,49 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/nightly/whatsnew.html" %}
+
+{% block page_css %}
+  {{ super() }}
+  {{ css_bundle('nightly_whatsnew_145') }}
+{% endblock %}
+
+{% set params = '?utm_source=firefox.com&utm_medium=referral&utm_campaign=wnp145nightly' %}
```

Comment:
firefox.com source is a little surprising given it's a mozilla.org WNP, but this has appeared in more places in current WNPs so again, just noting for the record, but fine to keep this way.

Reply:
(there's a redirection at sumo side anyways so they might be losing the referals at that point:/…)

Reply:
yes, I was on the fence about this, but went with exact copy doc